Sources/RegexBuilder/DSL.swift

```diff
@@ -15,9 +15,9 @@
 @available(SwiftStdlib 5.7, *)
 extension Regex {
   /// Creates a regular expression using a RegexBuilder closure.
-  public init<Content: RegexComponent>(
```

Comment:
Would this break ABI? Isn't `some RegexComponent<Output>` in the closure return type going to be an opaque return type, not a generic parameter on the parent declaration?

Reply:
This particular change is just a syntactic transformation of the original. You can get into trouble with multiple generic parameters, since the order is important to the ABI, but luckily all of those are `@_alwaysEmitIntoClient`. (This one should probably be that as well, tbh.)